src/views/Auth/Register/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react'
+import '../style.scss'
+import Input from 'components/Input'
+import Button from 'components/Button'
+import googleSvg from 'assets/google.svg'
+import registerSvg from 'assets/register.svg'
+import notepad from 'assets/notepad.svg'
+import { Link } from 'react-router-dom';
+
+export default class Auth extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {}
+    }
+
+    render() {
+        const logo = <img src={googleSvg} width="22" height="14" alt=" Google svg" /> 
+        return (
+        <div className="authWrapper">
+            <div className="auth">
+                <div className="auth__form">
+                    <h3 className="auth__form__brand"><span><img src={notepad} width="30" height="24" alt="Google svg" /> </span> O-notes</h3>
+                    <h1 className="auth__form__greeting">Register</h1>
+                    <Button logo={logo} text="Sign Up With Google" size='long'/>
+                    <p className='auth__form__or'>or</p>
+                    <form>
+                     <Input type="email" label="Email address"/>
```

Comment:
spacings between the Input components should be consistent with that of signup. So, add a `<br />` to the end of the Input components